### PR TITLE
Topic recognition (Improved UNet - 46281740)

### PR DESCRIPTION
**Pull request by Mimimitsu**
Name: Zichuan Huang
Student Number: 46281740

Hi, 

This pull request is a Tensorflow implementation of the improved UNet on the ISICs dataset.

It contains the following files:

- `README.md` The introduction of the Improved UNet and other information
- `dataset.py` Loading data
- `modules.py` The UNet model
- `driver_script.py` Training process of the model on the dataset
- `predict.py` Making a prediction
- `./image/*` The results of the prediction
- `./model/*` The .h5 file of the model

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
**This is an initial inspection, no action is required at this point**
- Code: OK
- Results: what is the test set dice? the plot is for validation
- Readme: should report qualitative results in readme; 
- Commit Messages:
- Other comments: please do not commit weights

**Comment by shakes76** [comment]
## Good Practice (Design/Commenting, TF/Torch Usage)
Adequate use and implementation
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage -2 (plots/testing missing)
Module present
Commenting
No Data leakage
Difficulty: Easy -10
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
ReadMe acceptable (could have had more info) -1
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
Feedback required (please remove weights and update request) -2
Request Description good
